.circleci/config.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       - run:
           name: Install sbt dependencies
           command: |
+            apt-get update && apt-get install -y git
```

Comment:
tioli, but elsewhere we run `apt-get install -y --no-install-recommends` so that it does less work. not sure if that helps here or not